ivy/func_wrapper.py

```diff
@@ -847,6 +848,7 @@ def _handle_out_argument(*args, out=None, **kwargs):
         """
         if out is None or is_compos_fn:
             return fn(*args, out=out, **kwargs)
+        handle_out_in_backend = handle_out_in_backend and not _is_variable(out)
```

Comment:
Given that we're not able to import `_is_variable` in this file, let's just move back to the way you were implementing it,
```
        if ivy.gradients._is_variable(out):
            handle_out_in_backend = False
```

Reply:
Sure 👍